x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/search_source_expression_form.tsx

```diff
@@ -172,10 +179,13 @@ export const SearchSourceExpressionForm = (props: SearchSourceExpressionFormProp
       from: `now-${timeWindow}`,
       to: 'now',
     });
-    testSearchSource.setField('filter', timeFilter);
+    testSearchSource.setField(
+      'filter',
+      timeFilter ? [timeFilter, ...ruleConfiguration.filter] : ruleConfiguration.filter
+    );
     const { rawResponse } = await firstValueFrom(testSearchSource.fetch$());
```

Comment:
qq @Dosant, unrelated to what we fix with this PR, but  this should be changed to `lastValueFrom`, right?